src/content/docs/es/tutorials/add-content-collections.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Tutorial - Ampliar con colecciones de contenido.
+description: >-
+  Convierte el código del tutorial de Crear un Blog cambiando el contenido de los archivos a colecciones de contenido.
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+import Option from '~/components/tutorial/Option.astro';
+
+Las **colecciones de contenido** son una forma poderosa de gestionar grupos de contenido similar, como lo son las publicaciones de un blog. Las colecciones ayudan a organizar tus documentos, validar el contenido de los archivos YAML y proveen seguridad de tipos con TypeScript automáticamente para todo tu contenido (incluso si no escribes nada de TypeScript por tu cuenta).
+
+<PreCheck>
+  - Mueve el directorio de publicaciones del blog en el directorio `src/content/`
+  - Crea un esquema que defina el contenido de la publicación en términos de frontmatter.
+  - Utiliza el método `getCollection()` para obtener el contenido de tus publicaciones y sus metadatos.
+</PreCheck>
+
+## Pre-requisitos
+
+Necesitarás **un proyecto existente de Astro que incluya archivos Markdown o MDX en el directorio  `src/pages/`**.
+
+Este tutorial utiliza [el proyecto de Crear un Blog terminado](https://github.com/withastro/blog-tutorial-demo) para demostrar cómo convertir un blog en colecciones de contenido. Puedes clonar y usar la base de código localmente, o completar el tutorial en tu navegador [editando el código del tutorial del blog en StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).
+
+Si lo prefieres, puedes seguir los pasos usando tus propios proyectos, pero, necesitarás realizar algunos ajustes en las instrucciones del código.
+
+Nosotros recomendamos usar nuestro proyecto de ejemplo para terminar este tutorial corto. Después, puedes utilizar lo aprendido para crear colecciones de contenido en tu proyecto.
+
+## Código de Construir un Blog
+
+En el [tutorial introductorio de Crear un Blog](/es/tutorial/0-introduction/), aprendiste acerca del [sistema de rutas basado en archivos](/es/core-concepts/routing/#rutas-estáticas): cualquier archivo `.astro`, `.md`, o `.mdx` dentro del directorio `src/pages/` se convertirá automáticamente en una página en tu sitio.
+
+Para crear tu primera publicación `https://example.com/posts/post-1/`, creaste el directorio `/posts/`y agregaste el archivo `post-1.md`. Después, agregaste un archivo Markdown en este directorio cada que querías agregar un nuevo publicación en tu blog.
```

Comment:
Todos los directorios no comienzan con `/` como lo hace `/posts/` aquí, y falta espacio antes de la y.

Reply:
Para el ejemplo del ejercicio, el directorio es `/posts/`, dado que el tutorial esta centrado en ése proyecto, considero que es mejor mantenerlo cómo esta.

Respect a la sugerencia de usar `cada vez`, agregarlo resulta redundante dado el contexto de la sentencia. Sigue teniendo mayor sentido `AS IS`.

Se agregó el espacio que hacía falta después de `/posts/`